ArgumentError: column name :Number_of_patients_in_ICU_on_a_ventilator_with_COVID_19 not found in the data frame
